drltrace/drltrace_win.config

```diff
@@ -0,0 +1,1170 @@
+# ***************************************************************************
+#
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice,
+#   this list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of Google, Inc. nor the names of its contributors may be
+#   used to endorse or promote products derived from this software without
+#   specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL GOOGLE, INC. OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+# DAMAGE.
+#
+
+# ***************************************************************************
+# drltrace default config file for library call arguments printing in Windows.
+
+# The syntax of this config file is simple. To add a new function, it's enough to specify
+# function's return type, function name and each argument of the function separated by
+# a pipe | symbol.
+# Example: int strcmp (char *, char *) -> int|strcmp|char *|char *
+# i#1948: The syntax now does not cover many other more complex function prototypes
+# e.g. atexit(). We need to improve that.
+#
+# NOTE: the syntax is not space sensitive.
+
+int|strcmp|char *|char *
+int|wcscmp|wchar *|wchar *
+int|printf|char *
+int|puts|char *
+
+# A group of exported functions from kernel32.dll
+VOID|AcquireSRWLockShared|++SRWLOCK*
+bool|ActivateActCtx|HANDLE|+ULONG_PTR*
+ATOM|AddAtomA|char*
+ATOM|AddAtomW|wchar*
+bool|AddConsoleAliasA|char*|char*|char*
+bool|AddConsoleAliasW|wchar*|wchar*|wchar*
+bool|AddIntegrityLabelToBoundaryDescriptor|++HANDLE*|SID*
+VOID|AddRefActCtx|HANDLE
+bool|AddSIDToBoundaryDescriptor|++HANDLE*|SID*
+bool|AddSecureMemoryCacheCallback|SECURE_MEMORY_CACHE_CALLBACK*
+VOID*|AddVectoredContinueHandler|ulong|VECTORED_EXCEPTION_HANDLER*
+VOID*|AddVectoredExceptionHandler|ulong|VECTORED_EXCEPTION_HANDLER*
+bool|AllocConsole|VOID
+bool|AllocateUserPhysicalPages|HANDLE|++ULONG_PTR*|+ULONG_PTR*
```

Comment:
Hmm, I assume you picked + and ++ to be short.  ++ is not very intuitive: +- seems a little better but OUT and INOUT or __out and __inout would be clearer and match conventions used in header files, including Microsoft's.  Would they push many of these over 90-char lines?  WDYT?